test/unit/auth/user-record.spec.ts

```diff
@@ -29,11 +29,12 @@ chai.use(chaiAsPromised);
 const expect = chai.expect;
 
 /**
+ * @param {string=} tenantId The optional tenant ID to add to the response.
  * @return {object} A sample valid user response as returned from getAccountInfo
  *     endpoint.
  */
-function getValidUserResponse(): object {
-  return {
+function getValidUserResponse(tenantId?: string): any {
```

Comment:
Use `{[key: string}: any}` instead of `any`.

Reply:
done